week3/teacher-live-coding/.env.example

```diff
@@ -1,15 +1,10 @@
 # Database - If you want to use your own database
-# DB_HOST = 127.0.0.1 
-# DB_USER = root 
+# DB_HOST = 127.0.0.1
+# DB_USER = root
 # DB_PASSWORD = db_password
 # DB_NAME = db_name
 # DB_PORT = 3306
 
-# Database already set up
-DB_HOST = eu-cdbr-west-03.cleardb.net
```

Comment:
It's bad practice to check these in, so I think we should avoid it.

Reply:
Ahh yes definitely. Slipped past me